lms/djangoapps/discussion_api/api.py

```diff
@@ -123,3 +127,78 @@ def get_thread_list(request, course_key, page, page_size):
 
     results = [ThreadSerializer(thread, context=context).data for thread in threads]
     return get_paginated_data(request, results, page, num_pages)
+
+
+def get_comment_list(request, thread_id, endorsed, page, page_size):
+    """
```

Comment:
We don't need the full-blown API doc stuff everywhere, agreed - I'm not
really concerned with adding more content but I do think
indentation benefits readability.

On Tuesday, May 19, 2015, Greg Price <notifications@github.com> wrote:

> In lms/djangoapps/discussion_api/api.py
> <https://github.com/edx/edx-platform/pull/8097#discussion_r30609554>:
>
> > @@ -123,3 +127,78 @@ def get_thread_list(request, course_key, page, page_size):
> >
> >      results = [ThreadSerializer(thread, context=context).data for thread in threads]
> >      return get_paginated_data(request, results, page, num_pages)
> > +
> > +
> > +def get_comment_list(request, thread_id, endorsed, page, page_size):
> > +    """
>
> The full Markdown-based format seems like overkill to me in cases where we
> are not planning to actually extract a docstring and include it in the
> published documentation. Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8097/files#r30609554>.
>